core/config/docs/core.toml

```diff
@@ -434,6 +434,32 @@ DeltaReconcile = '1m' # Default
 # but the host and port must be fully specified and cannot be empty. You can specify `0.0.0.0` (IPv4) or `::` (IPv6) to listen on all interfaces, but that is not recommended.
 ListenAddresses = ['1.2.3.4:9999', '[a52d:0:a88:1274::abcd]:1337'] # Example
 
+[Capabilities.Registry]
+# Enabled enables Registry
+Enabled = false # Default
```

Comment:
@bolekk Does this enable the registry or just syncing? The registry should always be enabled IMO, but there's an argument to be made about disabling syncing.

Reply:
I'll make it always-enabled.